docs/source/Installation.md

```diff
@@ -100,6 +100,14 @@ In the `env_file_app`, configure different variables as explained below.
 * `DJANGO_SECRET`: random 50 chars key, must be unique. If you do not provide one, Intel Owl will automatically set a new secret on every run.
 * `INTELOWL_WEB_CLIENT_DOMAIN` (example: `localhost`/`mywebsite.com`): the web domain of your instance, this is used for generating links to analysis results.
 
+
+### Deprecated environment configuration
+The following variables are deprecated and will be removed in the future.
+The new way to configure plugin secrets is to use the `Plugin Secrets` page in the GUI.
+If you had previously specified any variables in the environment, run `docker exec -ti intelowl_uwsgi python3 manage.py migrate_secrets`
+and then remove those secrets from the env file.
```

Comment:
super cool this. The only thing not stated here is the version number where these variables start to be deprecated 🤔  but obviously you cannot know that in advance...

Plus, this is also a breaking change in the end if we put in this way. 🤔  I would really like to avoid to do a 5.x release just for this change.

One cool thing would be to run this automatically in the uwsgi entrypoint _only_ if this is the first time that this is executed. How to check this? Maybe just trying to see whether there is any element in the PluginConfig table? If there is already anything, that would mean that it was already populated so we could skip that. Does it make sense to you?